config/vector.toml

```diff
@@ -12,37 +12,43 @@
 # Chat: https://chat.vector.dev
 # ------------------------------------------------------------------------------
 
+# Change this if you need to use a non-default directory for data storage:
 # data_dir = "/var/lib/vector"
 
-# Vector's API (disabled by default)
-# Enable and try it out with the `vector top` command
-[api]
-enabled = false
-# address = "127.0.0.1:8686"
-
-# Dummy Apache formatted logs
+# Random Syslog-formatted logs
 [sources.dummy_logs]
 type = "generator"
 format = "syslog"
 interval = 1
 
-# Parse Apache formatted logs
-# See the Remap reference: https://vector.dev/docs/reference/vrl/
+# Parse Syslog logs
+# See the Vector Remap Language reference for more info: https://vrl.dev
 [transforms.parse_logs]
 type = "remap"
+inputs = ["dummy_logs"]
 source = '''
-. = parse_syslog(.message)
+. = parse_syslog!(string!(.message))
 '''
 
-# Host-level metrics (cpu, memory, disk, etc)
-[sources.host_metrics]
-type = "host_metrics"
-
-# Vector's own internal metrics
-[sources.internal_metrics]
-type = "internal_metrics"
-
-# Print it all out for inspection
+# Print parsed logs to stdout
 [sinks.print]
 type = "console"
-inputs = ["parse_logs", "host_metrics", "internal_metrics"]
+inputs = ["parse_logs"]
+encoding.codec = "json"
+
+# Vector's GraphQL API (disabled by default)
+# Uncomment to try it out with the `vector top` command or
+# in your browser at http://localhost:8686
+#[api]
+#enabled = true
+#address = "127.0.0.1:8686"
+
+# Host-level metrics (CPU, memory, disk, etc.)
+# Uncomment this if you enable the API:
```

Comment:
I think `host_metrics` is useful for general node monitoring, but can you clarify what you intend by "Uncomment this if you enable the API"?

Reply:
👍 I think that's fine. I agree they make the console output overwhelming.